samples/peripheral/802154_phy_test/include/comm_proc.h

```diff
@@ -0,0 +1,84 @@
+/* Copyright (c) 2019, Nordic Semiconductor ASA
```

Comment:
Please use NCS license in all of the source files (`SPDX-License-Identifier: LicenseRef-Nordic-5-Clause`)

Reply:
Replaced all with proper license clause